ui/src/containers/Layout.js

```diff
@@ -38,6 +38,12 @@ const Layout = props => {
   }, [fetchClusterVersion]);
 
   const help = [
+    {
+      label: props.intl.messages.advanced_monitoring,
+      onClick: () => {
+        props.history.push('/grafana');
```

Comment:
May want to have this in the 'config' document as well, where we have `/api/kubernetes` etc. Otherwise, why have the config at all, and not hard-code the other paths as well?

Reply:
Yes good point!